### DISCUSSION TITLE
Installation question

### DISCUSSION THREAD

**Discussion by BumpaRoy** [Azure/data-api-builder#1695] [Q&A]
Trying to load DAB on another SQL Server (2019).  It does not have VS 2022 installed or IIS.  I have installed .Net 7.0 SDK, Runtime, Desktop Runtime and ASP.Net Core 7.0 Runtime.  I am still not able to get the DAB install to verify the version.  See screen print.
![2023-09-08_8-49-41](https://github.com/Azure/data-api-builder/assets/137801704/0e8ea8c9-0be1-4fc4-946d-bcc69420ba65)

**Comment by seantleonard** [comment]
You’ll need to install the .NET 6 SDK in order to install/build DAB.

**Reply by BumpaRoy** [reply]
ok, I am trying to document the steps on a fresh server.  Will that always be the case that the .Net 6 SDK is a pre-req?  Or will it eventually be supported in the current version?

**Reply by seantleonard** [reply]
.NET 6 is the most recent Long Term Support (LTS) version of .NET. https://dotnet.microsoft.com/en-us/platform/support/policy

We will be looking at .NET 8 once it reaches General Availability in a few months.
